ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/worldstate/NodeDataRequest.java

```diff
@@ -132,10 +107,33 @@ public NodeDataRequest setRequiresPersisting(final boolean requiresPersisting) {
   }
 
   public final void persist(final WorldStateStorage.Updater updater) {
+    if (pendingChildren.get() > 0) {
+      if (totalPending.incrementAndGet() > max + 999) {
```

Comment:
maybe we can explain why 999 ?

Reply:
I am going to remove it. maybe >= 1000 would make a better sense, I just update it when the difference is at least 1000